modules/templates/helper.go

```diff
@@ -97,6 +98,7 @@ func NewFuncMap() []template.FuncMap {
 		"SafeJS":        SafeJS,
 		"JSEscape":      JSEscape,
 		"Str2html":      Str2html,
+		"Md2html":       Md2html,
```

Comment:
We'd better avoid abbr names, `RenderMarkdown` or `Markdown2html` is clearer.

ps: the `Str2html` above has been discussed and we consider it is a bad example and should be refactored in future.

Reply:
`Markdown2html` it is then